e2e/pnpm_lockfiles/MODULE.bazel

```diff
@@ -10,6 +10,7 @@ PNPM_LOCK_VERSIONS = [
     "v60",
     "v61",
     "v90",
+    "v10",
```

Comment:
naming convention left something to be desired. Seems this should be v100 so you have a way to express v10.2